collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/SavePointTest.kt

```diff
@@ -225,6 +225,28 @@ class SavePointTest {
         rule.fillNewForm("two-question-audit.xml", "Two Question")
     }
 
+    @Test
+    fun whenBlankFormStartedThenSavedAndKilled_aSavepointShouldBeCreatedForASavedFormNotForTheBlankOne() {
```

Comment:
Might be good to link to the issue in a comment here as this is pretty specific.

Reply:
Done.